cds/modules/deposit/static/templates/cds_deposit/angular-schema-form/uiselect.html

```diff
@@ -21,6 +21,7 @@
                 {{select_model.selected.name}}
             </ui-select-match>
       <ui-select-choices
+        minimum-input-length="3"
```

Comment:
We can add something like [this](https://github.com/CERNDocumentServer/cds/blob/cdslabs_qa/cds/modules/deposit/static/json/cds_deposit/forms/video.json#L398) ``minLength`` in the options and use it like 

```
minimum-input-length="form.minLength || 3"
```